core/azure-core/src/main/java/com/azure/core/polling/PollResponse.java

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+import java.time.Duration;
+import java.util.Map;
+
+/**
+ * The life cycle of an operation depicted below
+ * 1. NOT-STARTED
```

Comment:
Same issue as the other JavaDoc list

Reply:
Will use <ul> . I will spend more time on documentation once the API is looking good.